modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -396,9 +396,18 @@ spec:
     - role: Contributor
 ----
 
-.. Remove the YAML file for any `CredentialRequest` object that is in Technology Preview or they cause the installation to fail. As of {product-version}, the only credential request in Technology Preview is for the `capi-operator`. To remove this request:
-To remove this request:
-... To list the credential request, run the following command:
+.. Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously. The secrets must be stored using the namespace and secret name defined in the `spec.secretRef` for each `CredentialsRequest` object. The format for the secret data varies for each cloud provider.
++
+[IMPORTANT]
+====
+The release image includes `CredentialsRequest` objects for Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set.
+
+If you are not using any of these features, do not create secrets for these objects. Creating secrets for Technology Preview features that you are not using can cause the installation to fail.
```

Comment:
conversely, _not_ creating these secrets if you are installing with tech-preview enabled can also cause the installation to fail.  All four cases:

* Tech-preview not enabled, you don't create manifests for the tech-preview secrets: success.
* Tech-preview not enabled, you do create manifests for the tech-preview secrets: failures like [namespace doesn't exist][1].
* Tech-preview enabled, you don't create manifests for the tech-preview secrets: failures like "hey, where's the secret I asked for?" from the tech-preview components.
* Tech-preview enabled, you do create manifests for the tech-preview secrets: success.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=2035903#c0

Reply:
I think we are ok wrt that - we are telling them to make secrets for every cred req *unless* they are not using the TP stuff. But I defer to others if we're worried about it :)

Reply:
With openshift/cloud-credential-operator#444, folks need to add `--enable-tech-preview` to their `ccoctl` calls (or take similar action if they are manually creating secrets without using `ccoctl`) to get secrets for every cred request that applies to their cluster.

Reply:
Ok, so I think this is two issues:
1. Need to update `ccoctl` process reqs step to note `--enable-tech-preview` flag use
2. "or take similar action if they are manually creating secrets" - do they need to do anything special other than what we already have? i.e. "Create YAML files for secrets in the `openshift-install` manifests directory that you generated previously."

Reply:
Revising as follows (will push change in a bit):
```
The release image includes `CredentialsRequest` objects for Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set.

If you are not using any of these features, do not create secrets for these objects. Creating secrets for Technology Preview features that you are not using can cause the installation to fail.

You can identify these objects by their use of the `release.openshift.io/feature-gate: TechPreviewNoUpgrade` annotation.
```